examples/Surface.nopadding.tsx

```diff
@@ -30,27 +30,27 @@ export default () => {
               padding: 0,
             }}
           >
-            <li style={listStyle}>
+            <li style={listStyle} tabIndex={0}>
```

Comment:
please don't add tabindex to everything

Reply:
to fix the scrollable violation, add `tabIndex={0}` only to `<Surface.Body>` and also add `as='article'` (so it has a sensible role) and `aria-labelledby` (with an id pointing to the `h2` element)

Reply:
Implemented this suggestion.